src/lib/helpers/config.helper.ts

```diff
@@ -19,5 +19,10 @@ export const LSP3_UP_KEYS = {
   LSP3_PROFILE: keccak256(toUtf8Bytes('LSP3Profile')),
 };
 
+export const ADDRESS_PERMISSIONS_ARRAY_KEY =
+  '0xdf30dba06db6a30e65354d9a64c609861f089545ca58c6b4dbe31a5f338cb0e3';
```

Comment:
Can you write the key as string next to it? as a comment, so it is easy to find what key name it is